ext/mbstring/tests/gh10648.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+GH-10648 (mb_check_encoding() returns true for incorrect but interpretable ISO-2022-JP byte sequences)
+--EXTENSIONS--
+mbstring
+--FILE--
+<?php
```

Comment:
I think all the test cases in this file should be repeated both for `JIS` and for `ISO-2022-JP`.